src/e2e/config/test-data/constants.js

```diff
@@ -0,0 +1,14 @@
+const expectedTableHeaders = [
```

Comment:
lets add this in the definition file only

Reply:
As per discussion, done this change